src/routes.tsx

```diff
@@ -12,12 +12,8 @@ const loadModule = ( cb ) => ( componentModule ) => {
 };
```

Comment:
cb needs annotation:

```
cb: (err:any, component: Component<any,any>)=>void
```
https://github.com/reactjs/react-router/blob/master/docs/API.md#getcomponentnextstate-callback

or check it in type definitions if there isn't a type for that already